### PR TITLE
Add Documentation for `registerIconPacks` in mermaid section (English / Japanese)

### PR DESCRIPTION
**Pull request by Take-Me1010**
Hi @shd101wyy

This PR relates https://github.com/shd101wyy/crossnote/issues/361 .

# What I did

- I added description for loading logos in mermaid in English / Japanese doc.

# What I did not

- other language doc suppport

### CODE REVIEW

**Comment by shd101wyy** [comment]
@Take-Me1010 thank you for the PR, could you please also help update the doc for `zh-cn`, `zh-tw`, and `ko-kr`. Below is what I got from chatgpt. I usually use it to help translate:

zh-cn
```
此外，您可以在 head.html 中注册图标标志（通过 Markdown Preview Enhanced: Customize Preview Html Head 命令打开），如下所示：
```

zh-tw
```
此外，您可以在 head.html 中註冊圖標標誌（通過 Markdown Preview Enhanced: Customize Preview Html Head 命令打開），如下所示：
```

ko-kr
```
또한 Markdown Preview Enhanced: Customize Preview Html Head 명령을 통해 열리는 head.html에서 아이콘 로고를 다음과 같이 등록할 수 있습니다:
```

**Review by shd101wyy — DISMISSED** [review]
PR approved. Please feel free to merge.

**Comment by Take-Me1010** [comment]
> @Take-Me1010 thank you for the PR, could you please also help update the doc for zh-cn, zh-tw, and ko-kr. Below is what I got from chatgpt. I usually use it to help translate:

Thaks for your comment, @shd101wyy .
I added descriptions for other language.

> PR approved. Please feel free to merge.

I have no write access, so could you merge this PR if you approve next?

**Review by shd101wyy — APPROVED** [review]

**Pull request merged**
